deploy/examples/cluster-test.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 spec:
   dataDirHostPath: /var/lib/rook
   cephVersion:
-    image: quay.io/ceph/ceph:v19
+    image: quay.ceph.io/ceph-ci/ceph:wip-pdonnell-68f47b56-debug-arm64 # ===== TESTING TESTING TESTING TESTING =====
```

Comment:
Anything you see with `# ===== TESTING TESTING TESTING TESTING =====` is intended to be removed when I am ready for the final review and I take the PR out of draft state. For now, I'm leaving these in place so it's easier for me to build/test and so others can do the same if they want.